keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepositoryTests.kt

```diff
@@ -50,14 +53,31 @@ internal class SqlVerificationRepositoryTests :
     clock = clock
   )
 
+  /**
+   * Generate a mock Spring environment that returns the default value for all boolean properties
+   */
+  private fun mockEnvironment() : Environment {
+    val defaultValue = slot<Boolean>()
+    val environment: Environment = mockk()
+
+    every {
+      environment.getProperty(any(), Boolean::class.java, capture(defaultValue))
+    } answers {
+      defaultValue.captured
+    }
+
+    return environment
+  }
```

Comment:
… it would really simplify all this stuff as you could just use `true` / `false`